shared/_push-notification-faq.mdx

```diff
@@ -0,0 +1,74 @@
+<!-- props:
+  linkTesting - Link to the Push Testing page
+  linkLogs - Link to the Push Logs page
+-->
+  
+
+## Delivery Condition Not Matched
+Most of the time, not receiving push notifications results from a use case that the API does not handle. Our current API implementation supports push notifications under specific cases:
+- The target user has at least one device registered with the Stream App.
+- The target user is a member of the channel that will send notifications.
+- The target user has **not muted** the channel the pushed message is from.
+- The target user has **not muted** the user that created the pushed message.
+- Push notifications are enabled in the channel type.
+- If using `thread`, the user is part of the thread that messages are sent in (i.e previously posted at least one message or were mentioned in that thread)
+- Message isn't sent with `skip_push=true` flag.
+
+If one of these conditions is not met, then the API will not trigger any push. As a result, you will not receive any push notification AND you won't see any data on this push notification attempt in the Dashboard Push Logs.
+
+## Device Registration
+At least one device must be registered for a specific user in order to receive push notifications. This is also a push delivery condition and a mandatory step in the push integration logic.
+
+One of the common reasons for not receiving a push notification on a device is the absence of a device token. Without this, a device cannot be registered on the Stream database.
+
+:::warning
+If this is the issue, it will not be reported in the <a href={props.linkLogs}>Dashboard Push logs</a>.
```

Comment:
It is the only way I have found to inject parameters from outside, where this component is being used.
I have 2 concerns about it:
1-. Props can only be used with html tags. On this example I couldn't use the "Markdown link syntax"
2-. The component could be used without this props and docusaurus doesn't complain about it. I have added a comment on top of the file with the props that needs to be used.

@petyosi is there any alternative? Those links will need to be dynamically depending on the SDK platform that is using it, and maybe every platform has different docusaurus structure